tempo_apio.py

```diff
@@ -44,5 +44,11 @@ def get_articles():
                     mimetype="application/json")
 
 
+@app.route('/caneton_stats')
+def get_caneton_stats():
+    s = db.stats.find({}, {'_id': False}).sort('datetime', pymongo.DESCENDING)
+    return jsonify(list(s))
```

Comment:
http://stackoverflow.com/a/12345646/5732783

I tested this and it removes the id completely. The response of the api endpoint on my instance:

```json
[
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:30:56.027322", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:25:55.064296", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:21:00.769815", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 1, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:16:01.428559", 
    "obamo_json_errors": 0
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:11:31.399178", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:06:06.391945", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 0, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 04:00:36.214950", 
    "obamo_json_errors": 1
  }, 
  {
    "articles_added": 18, 
    "articles_fetched": 55, 
    "datetime": "2017-03-26 03:56:22.269037"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 03:04:13.907514"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 02:52:28.782933"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 02:41:55.538538"
  }, 
  {
    "articles_added": 0, 
    "datetime": "2017-03-26 01:50:13.340902"
  }
]
```